packages/grafana-ui/src/components/Logs/getLogRowStyles.ts

```diff
@@ -1,10 +1,11 @@
 import { css } from '@emotion/css';
 import { GrafanaTheme2, LogLevel } from '@grafana/data';
-import { styleMixins, stylesFactory } from '../../themes';
+import { stylesFactory } from '../../themes';
+import { theme as fusebitTheme } from '../../themes/fusebit';
 
 export const getLogRowStyles = stylesFactory((theme: GrafanaTheme2, logLevel?: LogLevel) => {
   let logColor = theme.isLight ? theme.v1.palette.gray5 : theme.v1.palette.gray2;
-  const hoverBgColor = styleMixins.hoverColor(theme.colors.background.primary, theme);
+  const hoverBgColor = fusebitTheme.colors.background.primary;
```

Comment:
Why can't we have `theme` which is coming from GrafanaTheme2 be set to a `Fusebit` option, so that there isn't this code churn throughout the product?